src/test/java/de/tum/in/www1/artemis/util/CourseTestService.java

```diff
@@ -1620,6 +1620,7 @@ public void testGetExerciseStatsForCourseOverview() throws Exception {
         assertThat(courseDtos).hasSize(1);
         var dto = courseDtos.get(0);
         assertThat(dto.getCourseId()).isEqualTo(instructorsCourse.getId());
+        assertThat(dto.getActiveStudents().size()).as("course was only active for 3 days").isEqualTo(1);
```

Comment:
```suggestion
        assertThat(dto.getActiveStudents()).as("course was only active for 3 days").hasSize(1);
```

Reply:
This seems to be better to me, thanks for the suggestion.